src/main/java/core/basesyntax/dto/TransferAction.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.dto;
+
+public interface TransferAction {
```

Comment:
there is no need to make interface here - our dto is just a simple object, like fruit
the methods are getters